bin/nek

```diff
@@ -85,7 +85,7 @@ fi
 
 SESSION=${REAFILE%.rea}
 
-rm -f vtk/*
+#rm -f vtk/*
```

Comment:
Yes..all those are issues now, and I needed feedback and preferred options, etc, after members trying it out first. Also, the script has to be changed on other machines too.
We can begin with local machines first..